### ISSUE TITLE
Adding aria-invalid to search input doesn't work

### ISSUE THREAD

**Issue by hayatbiralem** [picocss/pico#173]
Maybe you deliberately ignored it to keep the CSS size small, but I wanted to point it out in case you overlooked it.

Produced issue on Codepen: https://codepen.io/hayatbiralem/pen/VwrJZva?editors=0100

Could be solved like this: https://codepen.io/hayatbiralem/pen/KKyjpeJ?editors=0100

I'll try to make a PR.

**Comment by lucaslarroche** [comment]
@hayatbiralem, 
Thank you.

It wasn't deliberate.
Support for RTL is still new and there are still quite a few tweaks missing here and there.
I will review your PR.
